### ISSUE TITLE
t.ENV.setPlatform called by tfjs@1.1.2 but not defined

### ISSUE THREAD

**Issue by ToonTalk** [tensorflow/tfjs#1778]
#1340 ## TensorFlow.js version
1.1.2.

#### Browser version
Version 75.0.3770.142 (Official Build) (64-bit)
#### Describe the problem or feature request

When running the following Snippet in Chrome Dev Tools I see the following in the console and yet the snippet works fine.

tfjs@1.1.2:2 webgl backend was already registered. Reusing existing backend factory.
t.registerBackend @ tfjs@1.1.2:2
(anonymous) @ tfjs:formatted:10442
(anonymous) @ tfjs:formatted:3
(anonymous) @ tfjs:formatted:4
tfjs@1.1.2:2 cpu backend was already registered. Reusing existing backend factory.
t.registerBackend @ tfjs@1.1.2:2
(anonymous) @ tfjs:formatted:16357
(anonymous) @ tfjs:formatted:3
(anonymous) @ tfjs:formatted:4
tfjs:formatted:16384 Uncaught TypeError: t.ENV.setPlatform is not a function
    at tfjs:formatted:16384
    at tfjs:formatted:3
    at tfjs:formatted:4

#### Code to reproduce the bug / link to feature request
Repeated execution of the following (works fine first time):

```
script = document.createElement('script');
script.src = "https://cdn.jsdelivr.net/npm/@tensorflow/tfjs";
document.body.appendChild(script);
script.onload = () => {
a = tf.data.array([1, 2, 3, 4, 5, 6, 7, 8])
        .shuffle(8)
        .map(n => n*n)
        .batch(5);
a.forEachAsync(e => e.print());
};
```

**Comment by rthadur** [comment]
@ToonTalk thank you, did you try latest tfjs version ?

**Comment by nsthorat** [comment]
I think you have have multiple copies of TensorFlow.js on the page with different versions that are stomping on one another.

**Comment by ToonTalk** [comment]
I did (unintentionally) load the same version multiple times (and then when reporting it I did try to switch to the version at https://cdn.jsdelivr.net/npm/@tensorflow/tfjs but the problem did occur before I mixed versions). Of course one shouldn't load TensorFlow.js more than once but if one does maybe one can be warned instead of this strange behaviour.

I'm unclear why when I closed the tab, opened a new tab the problem occured the first time I tried to load TensorFlow.js -- is there any TensorFlow.js state shared across tabs?
